examples/features_demo/code/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "hello-world"
+name = "features-demo"
```

Comment:
Not sure if we should really commit the lock files?

Reply:
Already thinking about ignoring outputs completely. Although I would like to have the examples complete without requiring installation of yarner, but it bothers me that rust binary examples complicate installation.

Reply:
See #69

Reply:
> it bothers me that rust binary examples complicate installation.

This is a non-issue once we publish to crates.io...